src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1258,17 +1336,30 @@ void TorrentHandle::forceRecheck()
     m_unchecked = false;
 
     if (isPaused()) {
+#if (LIBTORRENT_VERSION_NUM < 10200)
         m_nativeHandle.stop_when_ready(true);
+#else
+        m_nativeHandle.set_flags(libt::torrent_flags::stop_when_ready);
+#endif
         resume_impl(false);
     }
 }
 
-void TorrentHandle::setSequentialDownload(const bool b)
+void TorrentHandle::setSequentialDownload(const bool enable)
 {
-    if (b != isSequentialDownload()) {
-        m_nativeHandle.set_sequential_download(b);
-        m_nativeStatus.sequential_download = b; // prevent return cached value
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    m_nativeHandle.set_sequential_download(enable);
+    m_nativeStatus.sequential_download = enable;  // prevent return cached value
+#else
+    if (enable) {
+        m_nativeHandle.set_flags(libt::torrent_flags::sequential_download);
+        m_nativeStatus.flags |= libt::torrent_flags::sequential_download;  // prevent return cached value
```

Comment:
This line and below are introduced due to using `m_nativeStatus.flags` (instead of `m_nativeHandle.flags()`)

Reply:
I don't know either... the old code does it so I merely follows it.